gravitee-apim-portal-webui-next/src/app/catalog/catalog.component.ts

```diff
@@ -92,19 +98,33 @@ export class CatalogComponent {
   }
 
   public onFilterSelection(event: string) {
-    this.selectedFilter = event;
+    this.filter = event;
+    this.router.navigate([''], {
+      relativeTo: this.route,
+      queryParams: {
+        filter: this.filter === 'all' ? '' : this.filter,
```

Comment:
Instead of `this.filter`, you can use the parameter `event`.

Reply:
Done